src/libraries/System.Data.OleDb/src/OleDb_Util.NoCOMWrappers.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Data.OleDb
+{
+    internal static partial class ODB
+    {
+        internal static OleDbHResult GetErrorDescription(UnsafeNativeMethods.IErrorInfo errorInfo, OleDbHResult hresult, out string message)
```

Comment:
The code here between COMWrappers and NoCOMWrappers looks duplicated. Can it be folded back together?